src/util/qwlogging.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2024 WenHao Peng <pengwenhao@uniontech.com>.
+// SPDX-License-Identifier: Apache-2.0 OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+
+#pragma once
+
+#include <qwglobal.h>
+extern "C" {
+#include <wlr/util/log.h>
+}
+
+QW_BEGIN_NAMESPACE
+
+class QW_EXPORT QWLog {
+public:
+    static void init();
```

Comment:
Needs `QtMsgType` argument.

Reply:
And add an override function `static void init(wlr_log_importance, callbackFunction)`

Reply:
you mean  wlr_log_importance  There is no way QtMsgType can be used here